learn-more.html

```diff
@@ -1,13 +1,202 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Learn More</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>LEARN ZONE</title>
+  <link href='./style.css' rel='stylesheet' />
+  <meta charset="utf-8" />
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Aladin&family=Rock+Salt&display=swap');
+    @import url('https://fonts.googleapis.com/css2?family=BenchNine:wght@300&family=Pathway+Gothic+One&display=swap');
+  </style>
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+
+    <!--- start main grid-->
+
+  <div class="learnmorecontainer">
+
+    <!--- start header !-->
+
+    <div class="sidebar-blank1"></div>

```

Comment:
Ive seen a handful of these `class='sidebar-blank1'` elements. There are better ways to create placeholders inside of your grid container that wouldn't add bloat to your HTML file